docs/organizations-and-projects.mdx

```diff
@@ -0,0 +1,15 @@
+# Organizations and Projects
+
+## Organizations
+Organizations are the top-level umbrella for all the projects in a company. A user can create multiple organizations to separate projects under different companies or teams.
```

Comment:
I think 

> Organizations are the top-level umbrella for all the projects in a company.

is opinion. We recommend this, but it doesn't have to be that. I think I would start with the general description, something like: "Organizations are collections of projects and where permissions for users are granted. We recommend to create an organization per company."

Reply:
Fixed this.